storage/src/mmr/mod.rs

```diff
@@ -72,6 +72,6 @@ use thiserror::Error;
 
 #[derive(Error, Debug)]
 pub enum Error {
-    #[error("a required element has been pruned: {0}")]
-    ElementPruned(u64),
+    #[error("an element required for this operation has been pruned")]
```

Comment:
Dropped the u64 parameter here because we are moving to a world where we'll know that some element required to generate a proof is missing but without knowing exactly which one it is. (https://github.com/commonwarexyz/monorepo/issues/459)

Reply:
Thanks for the context!